packages/fxa-auth-server/lib/routes/recovery-codes.js

```diff
@@ -188,6 +188,7 @@ module.exports = (log, db, config, customs, mailer) => {
             uaOSVersion: ua.osVersion,
             uaDeviceType: ua.deviceType,
             uid,
+            numberRemaining: remaining,
```

Comment:
nit: the rest of the keys appear to be in alpha order.

Reply:
Good catch, updated!